src/components/navigation/Sidebar/Sidebar.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../../styles/variables.scss';
+@import '../../../styles/mixins.scss';
+
+.dc-sidebar {
+    $transitionTime: 0.4s;
+
+    @include mobile-tablet-only();
+
+    position: fixed;
+    top: var(--dc-header-height, #{$headerHeight});
+    // TODO: delete
```

Comment:
there are a lot of unimplemented tudos and commented code in this PR, it is worth fixing